tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -582,50 +582,50 @@ func getLimits(memorySet bool, logSizeSet bool, timeoutSet bool, memory int, log
     return limits
 }
 
-func nestedError(errorMessage string, err error) (error) {
+func nestedError(errorMessage *string, err *error) (error) {
     return whisk.MakeWskErrorFromWskError(
-        errors.New(errorMessage),
+        errors.New(*errorMessage),
         err,
         whisk.EXITCODE_ERR_GENERAL,
         whisk.DISPLAY_MSG,
         whisk.DISPLAY_USAGE)
 }
 
-func nonNestedError(errorMessage string) (error) {
+func nonNestedError(errorMessage *string) (error) {
     return whisk.MakeWskError(
-        errors.New(errorMessage),
+        errors.New(*errorMessage),
         whisk.EXITCODE_ERR_USAGE,
         whisk.DISPLAY_MSG,
         whisk.DISPLAY_USAGE)
 }
 
-func actionParseError(cmd *cobra.Command, args []string, err error) (error) {
-    whisk.Debug(whisk.DbgError, "parseAction(%s, %s) error: %s\n", cmd, args, err)
+func actionParseError(cmd *cobra.Command, args []string, err *error) (error) {
+    whisk.Debug(whisk.DbgError, "parseAction(%s, %s) error: %s\n", cmd, args, *err)
 
     errMsg := wski18n.T(
         "Invalid argument(s). {{.required}}",
         map[string]interface{}{
             "required": err,
         })
 
-    return nestedError(errMsg, err)
+    return nestedError(&errMsg, err)
 }
 
-func actionInsertError(action *whisk.Action, err error) (error) {
-    whisk.Debug(whisk.DbgError, "client.Actions.Insert(%#v, false) error: %s\n", action, err)
+func actionInsertError(action *whisk.Action, err *error) (error) {
+    whisk.Debug(whisk.DbgError, "client.Actions.Insert(%#v, false) error: %s\n", action, *err)
 
     errMsg := wski18n.T(
-        "Unable to create action '{{.name}}': {{.err}}",
+        whisk.UNABLE_CREATE_ACTION_ERR,
```

Comment:
this change does not seem related to the rest of the commit changes.  this is probably best moved into another pr focusing on using constants as labels for all pii strings